client/extensions/woocommerce/app/index.js

```diff
@@ -89,10 +95,14 @@ function mapStateToProps( state ) {
 	const canUserManageOptions =
 		( siteId && canCurrentUser( state, siteId, 'manage_options' ) ) || false;
 	const isAtomicSite = ( siteId && !! isSiteAutomatedTransfer( state, siteId ) ) || false;
+	const hasPendingAutomatedTransfer =
+		( siteId && !! hasSitePendingAutomatedTransfer( state, siteId ) ) || false;
```

Comment:
`hasSitePendingAutomatedTransfer` will return `null` or `false` if `siteId` is `null` or in any other bad situation so we don't need all the code around the function call.

Not a blocker. :)